CMakeLists.txt

```diff
@@ -643,6 +639,12 @@ if (WIN32 OR is_xcode OR unity)
 
   list(APPEND src ${beast_unity_srcs} ${ripple_unity_srcs})
 
+  add_with_props(src/unity/test_unity.cpp
```

Comment:
@bachase Since both `test_unity.cpp` and `nodestore.cpp` use exactly the same properties, I would use the same `add_properties` call to add them.

The only way I know to do that is by assigning the two files to an intermediate variable and feeding that variable the the function (something like `add_with_props("${unity_nodestore_srcs}" `). There may be a better way to create the list inline, but I don't know it.